liubov.tarasenko/homework7/image.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'pry'
+require 'faraday'
+
+class Image
+  def download(url)
+    @url = url
+    ext = @url.split('.').last
```

Comment:
Extension will not be valid if provided url contains params like
`https://miro.medium.com/max/4000/1*KUy_KKExZrSpBuv9XfyBgA.png?somasda=asd&location=Cherkassy`

Reply:
do we consider it as critical?

Reply:
Extension should be taken from content type as any image will have a content type in format of `image`/`extension`

Reply:
done